app/main/routes.py

```diff
@@ -708,3 +711,179 @@ def terms_of_use():
 @bp.app_errorhandler(HTTPException)
 def http_exception(error):
     return render_template(f"{error.code}.html"), error.code
+
+
+@bp.route("/generate_manifest/<uuid:record_id>")
+@access_token_sign_in_required
+def generate_manifest(record_id: uuid.UUID):
```

Comment:
does this need to be a an endpoint if we only need the response in the `record` route? If not, we could just make it a function?

Reply:
ah Im just piecing it all together now I think. Do you have documentation that explains the manifest and file url generation?

Reply:
I think it might be useful to chat through this, as Im wondering whether we should have a separate api for this manifest, or somehow make it private as I feel we shouldnt be exposing the file url or manifest to the users of the application if possible

Reply:
Discussed on call on why this route is required for Universal Viewer. 
- UV requires a URL to access the manifest file from in JSON format
- Manifest data can be created and is behind a protected route